pkgs/applications/video/jellyfin-media-player/default.nix

```diff
@@ -0,0 +1,119 @@
+{ lib
+, fetchFromGitHub
+, fetchpatch
+, fetchzip
+, mkDerivation
+, stdenv
+, Cocoa
+, CoreAudio
+, CoreFoundation
+, MediaPlayer
+, SDL2
+, cmake
+, libGL
+, libX11
+, libXrandr
+, libvdpau
+, mpv
+, ninja
+, pkg-config
+, python
+, qtbase
+, qtwayland
+, qtwebchannel
+, qtwebengine
+, qtx11extras
+}:
+
+mkDerivation rec {
+  pname = "jellyfin-media-player";
+  version = "unstable-2021-04-13";
+
+  src = fetchFromGitHub {
+    owner = "iwalton3";
+    repo = "jellyfin-media-player";
+    rev = "e161d06f8a04fe2f36e1ae130378a37be1e264bc";
+    sha256 = "sha256-W2EeeStlpg1sHHStE5XOTCfviaabxRX/NfSoZU0CVPI=";
+  };
+
+  jmpDist = fetchzip {
+    url = "https://github.com/iwalton3/jellyfin-web-jmp/releases/download/jwc-1.7.1/dist.zip";
+    sha256 = "sha256-RIk+ltTyHqkMT7zhGwHf/4g6WPUT+H9u/05weUnRc3U=";
+  };
+
+  patches = [
+    # Fix hwdec for wayland (https://github.com/iwalton3/jellyfin-media-player/pull/23)
+    (fetchpatch {
+      url = "https://github.com/iwalton3/jellyfin-media-player/commit/6ae471023a30b1f16d2ed54bf48717af1483352e.patch";
+      sha256 = "sha256-736xpCJ7L1f8oX7HOMYCFzbzC2ZnH6tXUdw1y/fPG4w=";
+    })
+  ] ++ lib.optionals stdenv.isDarwin [
```

Comment:
Does the patch break linux? if not, please apply it uncondional.

Reply:
Nope, doesn't impact the linux build - removed the conditional.